.circleci/config.yml

```diff
@@ -167,7 +167,7 @@ jobs:
       WORKDIR: "workspace/test_linux_vcs_py2"
       OS: "linux-64"
       PY_VER: "py2"
-      CUSTOM_CHANNELS: ""
+      CUSTOM_CHANNELS: "-c danlipsa"
```

Comment:
I noticed you said temporarily pulling from danlipsa channel in the commit. Do you still need these channels?

Reply:
I only needed them to have circleci pull the new VTK which is needed for the MR. After the new VTK is built and uploaded they are not needed anymore. The new packages can be uploaded from my channel, or better they can be rebuild by @muryanto1 Note that several packages need to be rebuilt.

Reply:
@danlipsa I understand we need to rebuild vtk-cdat to upgrade vtk. 
Why do we need to rebuild libcdms, cdtime, cdms2? Thanks,

Reply:
This is because you cannot have binary libraries build at different times in your list of packages. Once you need to rebuild one, you need to rebuild all of them, otherwise they will depend on different versions of the same libraries so you'll get conflicts.